packages/appstore/src/components/modals/cfd-password-modal/cfd-password-modal.tsx

```diff
@@ -8,7 +9,16 @@ import { localize, Localize } from '@deriv/translations';
 import { useStores } from 'Stores';
 import './cfd-password-modal.scss';
 
-const CFDPasswordForm = ({ ...props }) => (
+type TProps = {
+    validatePassword: any;
+    submitMt5Password: any;
+    is_real_financial_stp: boolean;
+    closeModal: any;
+    form_error?: any;
+    account_title: any;
+};
+
+const CFDPasswordForm = ({ ...props }: TProps) => (
```

Comment:
```suggestion
const CFDPasswordForm = ({ ...props }: TProps) => (
```
De-structure the props according to the type declaration. Like the above use a more specific naming for the typings. Eg; `CFDPasswordFormProps`

Reply:
Done